tools/please_shim/main.go

```diff
@@ -71,6 +71,26 @@ func parseFlags() (*flags.Parser, error) {
 	return parser, nil
 }
 
+func setLogging() {
+	verbosity := cli.MinVerbosity
+	if opts.OutputFlags.ShimDebugVerbosity {
+		verbosity = cli.MaxVerbosity
+		os.Args = filterArgsFlag(os.Args, "--shim_debug_verbosity")
```

Comment:
This is required otherwise the main binary will throw a fatal error at the unknown flag. Having this flag as boolean simplifies the filtering too.